### PR TITLE
Azure sig

### PR DESCRIPTION
**Pull request by jensh007**
**How to categorize this PR?**

/kind enhancement
/area os
/os garden-linux

**What this PR does / why we need it**:
ci: add support for Azure image galleries

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user

```

### CODE REVIEW

File: ci/promote.py [hunk 1]
```diff
@@ -364,20 +404,30 @@ def main():
...
-        ),
-        version_str=version,
-        build_targets=parsed.build_targets,
-        cicd_cfg=cicd_cfg,
-        flavour_set=flavour_set,
-        build_type=parsed.target,
-    )
+    if parsed.azure_sig:
+        filtered = [release for release in releases if release.platform == 'azure']
+        release = filtered[0] # there should be only onme
```
Comment (AndreasBurger) [inline ci/promote.py:409]:
```suggestion
        release = filtered[0] # there should be only one
```

**Review by AndreasBurger — APPROVED** [review]
lgtm (if rebased)

**Review by gehoern — CHANGES_REQUESTED** [review]
Please Rebase

File: ci/promote.py (hunk 1 above)
Comment (jensh007) [reply ci/promote.py]: done

**Review by gehoern — APPROVED** [review]

**Pull request merged**